zone.go

```diff
@@ -182,6 +198,58 @@ func (z *Zone) findLabels(s string, targets []string, qts qTypes) (*Label, uint1
 			}
 		}
 	}
+	// check against each glob label
+	var found bool
```

Comment:
This is just duplicating the same logic elsewhere, right? That doesn't seem right.  (But might be because of the glob vs wildcard feature?)

Reply:
That's right. I don't like it too, it could've been more elegant. However, I chose to repeat (duplicate) the code for clarity.